include/circt/Support/LoweringOptions.h

```diff
@@ -151,6 +151,9 @@ struct LoweringOptions {
   /// If true, emit a comment wherever an instance wasn't printed, because
   /// it's emitted elsewhere as a bind.
   bool emitBindComments = false;
+
+  /// If true, do not emit a version comment at the top of each verilog file.
+  bool doNotEmitVersionComment = false;
```

Comment:
We try to avoid negatives here (but haven't been perfectly consistent, c.f. disallowExpressionInliningInPorts).  Please use emitVersionComment.

Reply:
See this comment: https://github.com/llvm/circt/pull/4811#issuecomment-1465321751

I think by default we want this on to avoid everyone having who is expecting this to explicitly enable this. I agree the negative is not great, but it seems like a better option than changing the default opaquely?

Reply:
`omitVersionComment` would satisfy both right?

Reply:
done! now `omitVersionComment`. thanks for the feedback all :)